packages/dev/core/src/Maths/math.scalar.ts

```diff
@@ -15,7 +17,7 @@ export class Scalar {
      * @returns true if the absolute difference between a and b is lower than epsilon (default = 1.401298E-45)
      */
     public static WithinEpsilon(a: number, b: number, epsilon: number = 1.401298e-45): boolean {
-        return Math.abs(a - b) <= epsilon;
+        return true;

```

Comment:
Instead of implementing them and then removing the implementation as side effect, you could assign it directly. Will save a bit of code:

``` 
public static WithinEpsilon:  (a: number, b: number, epsilon: number = 1.401298e-45) => boolean = WithinEpsilon
```
Same goes to the rest.

This will also help us being side-effects free (or more free? :-))

Reply:
yeah you are right this is far better!